### PR TITLE
Added operations for * and /

### PR DESCRIPTION
**Pull request by AyanMondal7m**
Operations added for division and multiplication

### CODE REVIEW

**Review by nischithas96 — APPROVED** [review]
Looks good

File: src/calculator.js [hunk 1]
```diff
@@ -109,7 +109,16 @@ class Calculator {
 				break;}
 			case '-':{
 				result=operand1-operand2;
-				break;}
+				break;
+			}
+			case '*': {
+				result = operand1 + operand2;
```
Comment (2020mt93512) [inline src/calculator.js:115]:
```suggestion
				result = operand1 * operand2;
```

File: src/calculator.js [hunk 2]
```diff
@@ -109,7 +109,16 @@ class Calculator {
...
+				break;
+			}
+			case '/': {
+				result = operand1 - operand2;
```
Comment (2020mt93512) [inline src/calculator.js:119]:
```suggestion
				result = operand1 / operand2;
```

Comment (2020mt93512) [inline src/calculator.js:119]: Maybe we can add a divide by zero check also

**Review by 2020mt93512 — CHANGES_REQUESTED** [review]

**Review by 2020mt93512 — APPROVED** [review]

**Review by tanya93574 — APPROVED** [review]

**Pull request merged**